_chapters/chapter_02.md

```diff
@@ -177,9 +202,9 @@ layout: rails_tutorial
       my_favorite_book = Book.new
       ```
 
-      This will create a new instance of `Book` that we can refer to as `my_favorite_book`.
+      This just assigned a new instance of `Book` to a [variable](http://tutorials.jumpstartlab.com/projects/ruby_in_100_minutes.html#2.-variables) so that we can refer to it as `my_favorite_book`.
```

Comment:
Instead of "This will create" or "This just assigned", what if we said "This assigns".